### PR TITLE
How to Rename a Local Branch

### PR DESCRIPTION
**Pull request by ArmenAvetisyan**
New Content for the Wiki #367

### CODE REVIEW

**Comment by bugron** [comment]
Արմեն, շնորհակալ եմ: Որ խնդրեմ, չե՞ք նշի նաև `git branch -M` տարբերակի մասին (հարկադրաբար անվանափոխում):

**Comment by Rafase282** [comment]
Thanks @bugron also we are welcoming articles in other languages if @armenavetisyan is interested. However, I'm considering whether to limit those to languages someone form the team can understand for moderation purposes or to just rely on online translators as to not avoid restrictions.

File: rename-local-branch.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# How to Rename Local Branch
```
Comment (batjko) [inline rename-local-branch.md]: "How to Rename **A** Local Branch"

File: rename-local-branch.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+Rename <b>any</b> branch:
```
Comment (batjko) [inline rename-local-branch.md]: This should probably be enhanced as a sub-header, say with 
```md
### Rename any branch
```

**Pull request closed without merging**